src/components/OSCALCatalogGroup.js

```diff
@@ -30,20 +32,58 @@ const StyledControlDescriptionWrapper = styled("div")`
 
 function CollapseableListItem(props) {
   const [open, setOpen] = React.useState(false);
+  const [listItemOpened, setListItemOpened] = React.useState(false);
 
   const handleClick = () => {
     setOpen(!open);
   };
 
+  const handleFragment = useCallback(() => {
+    if (!listItemOpened) {
+      const { hash } = window.location;
+      // Ensure fragment exists and grab element associated
+      const controlFragment = hash?.substring(1);
+      const controlGroupingFragment =
+        determineControlGroupFromFragment(controlFragment);
+      // Locate the element with the provided fragment and scroll to the item
+      if (controlGroupingFragment) {
+        // Find control list state and open collapsable item
+        if (controlFragment.includes(props.itemText?.props?.value)) {
```

Comment:
I'd really rather we not fetch the `props` of a passed Component... This feels very hacky and is going to become rather complicated when we need to start adding type information I fear.

Is there a better way we can represent this?